### PR TITLE
{tools,vis}[foss/2018a] GraphicsMagick v1.3.28, Ghostscript v9.22

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
depends on ~~#5979~~ (cairo, GLib)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5e72837224115759322f1d28bf72e4ca for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2105.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/471405da0f139301f0bd38ae2bb234fc for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/6e92e9fbbe57b8b0405464bb28f004f0 for a full test report.

File: easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-GCCcore-6.4.0-cairo-1.14.12.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    # use same binutils version that was used when building GCCcore toolchain
+    ('binutils', '2.28'),
+]
+
+
+# Do not use local copies of zlib, jpeg, freetype, and png
+preconfigopts = "mv zlib zlib.no && mv jpeg jpeg.no && mv freetype freetype.no && mv libpng libpng.no && "
+preconfigopts += 'export LIBS="$LIBS -lz" && '
+
+configopts = "--with-system-libtiff --enable-dynamic"
```
Comment (verdurin) [inline easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.22-GCCcore-6.4.0-cairo-1.14.12.eb]: @boegel Missing sanity check?

File: easybuild/easyconfigs/g/GraphicsMagick/GraphicsMagick-1.3.28-foss-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ('libpng', '1.6.34'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('LibTIFF', '4.0.9'),
+    ('libxml2', '2.9.7'),
+    ('XZ', '5.2.3'),
+    ('zlib', '1.2.11'),
+    ('Ghostscript', '9.22', '-cairo-1.14.12'),
+]
+
+modextrapaths = {'CPATH': ['include/GraphicsMagick']}
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GraphicsMagick/GraphicsMagick-1.3.28-foss-2018a.eb]: Missing sanity check here, too.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2672.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f9a13a55aeb92fff30a2996badcf01da for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2025.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/a462334939326f96738484187227a415 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.4.1708, Intel Xeon E312xx (Sandy Bridge), Python 2.7.5
See https://gist.github.com/86a09d64933342ce45d38916d5ec3df5 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**